Dockerfile

```diff
@@ -14,7 +14,7 @@ COPY debezium-connector-postgres/target/debezium-connector-postgres-*.jar $KAFKA
 ENV KAFKA_OPTS="-Djdk.tls.client.protocols=TLSv1.2"
 
 # Add the required jar files to be packaged with the base connector
-RUN cd $KAFKA_CONNECT_PLUGINS_DIR/debezium-connector-postgres && curl -sLo kafka-connect-jdbc-10.6.5.jar https://github.com/yugabyte/kafka-connect-jdbc/releases/download/10.6.5-CUSTOM/kafka-connect-jdbc-10.6.5.jar
+RUN cd $KAFKA_CONNECT_PLUGINS_DIR/debezium-connector-postgres && curl -sLo kafka-connect-jdbc-10.6.5.jar https://github.com/yugabyte/kafka-connect-jdbc/releases/download/10.6.5-CUSTOM.1/kafka-connect-jdbc-10.6.5-CUSTOM.1.jar
```

Comment:
why do  we need this change?

Reply:
This is for the jar where we're printing all the messages being written to the target database.